specification/deviceprovisioningservices/data-plane/Microsoft.Devices/preview/2021-11-01-preview/examples/device/CertificateIssuance/issueCertificate.json

```diff
@@ -0,0 +1,26 @@
+{
+  "parameters": {
+    "api-version": "2021-11-01-preview",
+    "idScope": "contoso-id-scope",
+    "registrationId": "contoso-registration-id",
+    "request": {
+      "certificateType": "serverCertificate",
+      "csr": "-----BEGIN CERTIFICATE REQUEST-----MIIBJzCBzQIBADAvMS0wKwYDVQQDDCQ3ZDVhZmNiMC04NGI4LTQyYzYtYWJkYS0yYTQ0MTI3N2Q0MzMwWTATBgcqhkjOPQIBBggqhkjOPQMBBwNCAARBafB5bAi8DKq8F0UFsuEy+y6AHpk8YvIGE5g4ReeZIOSQ30obp4tFB7IdJQawTq3YKp1zOv5c8G19aykuGosSoDwwOgYJKoZIhvcNAQkOMS0wKzAMBgNVHRMBAf8EAjAAMBsGA1UdEQQUMBKCCmNvbnN0by5jb22HBAoAAAEwCgYIKoZIzj0EAwIDSQAwRgIhAKgIRbM0o4dWuXaQfJAD1WzM+FtowB3zRUAnrGYdRHJmAiEA/NzniwfySRO8/6Fr04y24ZesbqwWET8OOXuRn7KPlTk=-----END CERTIFICATE REQUEST-----"
+    }
+  },
+  "responses": {
+    "202": {
+      "description": "HTTP 202 (Accepted) if the operation was successfully started and will complete asynchronously.",
+      "headers": {
+        "retry-after": 10,
```

Comment:
10 should be string

Reply:
Thanks this worked for me. I would like to note that retry-after is declared as an integer, can you explain why this should be declared as a string for it succeed validation?

           "headers": {
              "retry-after": {
                "description": "The retry interval in seconds.",
                "type": "integer",
                "format": "int32"
              },

Will this be a breaking change for our existing clients? The tooling here will not tell me that since there is no prior swagger spec that was submitted here.

Reply:
no, this is not breaking change